_posts/2025-08-14-walkthrough-and-welcome-tab.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Introducing JabRef's New Walkthrough and Welcome Tab
+tags: [gsoc, walkthrough, welcome-tab]
+author: Yubo-Cao
```

Comment:
you need to add your name to the authors list https://github.com/JabRef/blog.jabref.org/blob/main/_data/authors.yml